0/rkm79791/p0.py

```diff
@@ -0,0 +1,96 @@
+#This is the first code I've ever written in python... maybe it works! 
+<<<<<<< HEAD
+#Ryan Maynard
+=======
+#By Ryan Maynard
+>>>>>>> eb020fabe0d3c916959d7f72da6bd4165d494b37
```

Comment:
This header, which was probably automatically generated through Git, makes the script fail with Python 2.7. We should look into why Git is automatically generating this header.

Reply:
This looks like an attempt to resolve a merge conflict gone wrong. Check what sequence of git commands you used to commit and submit these. Did you deal with multiple branches, change commit history...?